scripts/stf/filters.json

```diff
@@ -0,0 +1,43 @@
+[
+    "CustomLabel.npe03__RecurringDonationInstallmentPeriodMonthly",
+    "CustomLabel.npe03__RecurringDonationInstallmentPeriodQuarterly",
+    "CustomLabel.npe03__RecurringDonationInstallmentPeriodWeekly",
+    "CustomLabel.npe03__RecurringDonationInstallmentPeriodYearly",
+    "CustomLabel.npe03__RecurringDonationMultiplyValue",
+    "PicklistValue.Account.npo02__SYSTEM_CUSTOM_NAMING.Name",
+    "PicklistValue.Contact.npe01__PreferredPhone.Home",
+    "PicklistValue.Contact.npe01__PreferredPhone.Mobile",
+    "PicklistValue.Contact.npe01__PreferredPhone.Other",
+    "PicklistValue.Contact.npe01__PreferredPhone.Work",
+    "PicklistValue.Contact.npe01__Preferred_Email.Alternate",
+    "PicklistValue.Contact.npe01__Preferred_Email.Personal",
+    "PicklistValue.Contact.npe01__Preferred_Email.Work",
+    "PicklistValue.Contact.npe01__SystemAccountProcessor.One-to-One",
+    "PicklistValue.Contact.npo02__SystemHouseholdProcessor.All Individual Contacts",
+    "PicklistValue.Contact.npo02__SystemHouseholdProcessor.All New or Edited Contacts",
+    "PicklistValue.Contact.npo02__SystemHouseholdProcessor.No Contacts",
+    "PicklistValue.Opportunity.npsp__Matching_Gift_Status.Potential",
```

Comment:
As per your Chatter post, I have removed the 3 Matching_Gift_Status picklist translations from the filters list.